extensions/ql-vscode/src/interface.ts

```diff
@@ -400,6 +400,7 @@ export class InterfaceManager extends DisposableObject {
       }
     );
     const resultSet = transformBqrsResultSet(schema, chunk);
+    results.setResultCount(resultSet.schema.rows);
```

Comment:
This is a little confusing. The `alerts` table has 33 results in this example, but the `#select` table has 3906 results. I think people would expect that the result count matches the number of rows in the default table (in this case, it's `alerts`).

I'm not entirely sure why there is a difference here. I'm just running the example query from the starter repo:

```ql
/**
 * @name Empty block
 * @kind problem
 * @problem.severity warning
 * @id javascript/example/empty-block
 */

import javascript

from BlockStmt b
where b.getNumStmt() = 0
select b, "This is an empty block."
```

It would be nice if we could get the counts for the default table, but I'm not sure how easy this is without loading the sarif.

<img width="742" alt="_Extension_Development_Host__-_CodeQL_Query_Results_—_vscode-codeql-starter__Workspace_" src="https://user-images.githubusercontent.com/363559/110833060-dc4d9a80-8250-11eb-831f-321e38e2fc31.png">